src/css/application.css

```diff
@@ -477,6 +486,50 @@ body {
   white-space: normal;
 }
 
+/** @define project-picker */
+
+.project-picker__nav {
+  display: flex;
+  border-bottom: 1px solid var(--color-chrome);
+  list-style: none;
+  padding-left: 0;
+  margin-bottom: 0;
+}
+
+.project-picker__nav-tab {
+  padding: 0.75rem 2rem;
+  border: 1px solid transparent;
+  border-top-left-radius: 0.25rem;
+  border-top-right-radius: 0.25rem;
+  margin-bottom: -1px;
+  cursor: pointer;
+}
+
+.project-picker__nav-tab-active {
+  border-color: var(--color-chrome) var(--color-chrome) #fafafa;
```

Comment:
Is there no existing gray we can use in place of the ad-hoc `#fafafa`?

Reply:
Yah I am using what was defined on the `.modal__contents` selector. Would it make sense to pull this out into a variable or change the `.modal__contents` background-color to something predefined.